app/controllers/users_controller.rb

```diff
@@ -0,0 +1,33 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
+
+    if @user.save
+      redirect_to new_user_path
+    else
+      redirect_to users_path
+    end
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update(user_params)
+      redirect_to users_path
+    else
+      render :edit
+    end
+  end
+
+  def user_params
+    params.require(:user).permit(:username, :email, :password)
```

Comment:
That route doesn't exists
I got this error when update an user
![Screenshot at 2020-09-30 17:43:26](https://user-images.githubusercontent.com/8848104/94717588-79261680-0348-11eb-9aa1-caf143806c39.png)

Redirect to `edit_user_path`